### PR TITLE
4.2 docs finished

### PR DESCRIPTION
**Pull request by weeryan17**
Added Info on things walshy added into 4.1 but never had an official changelog
Expanded permissions to include rename and clone

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -6,12 +6,24 @@
...
+	 - Clone and rename have been added for groups
  - Buttons!
 	 - `_song` and `_skip` have buttons now. More to come in the future
 	 - Pages now have buttons for easy navigation.
  - Pages
 	 - They look nicer.
 	 - Some back-end work.
+ - Nino
+	 - It's an anti invite system for your server
+	 - You can find everything you need for nino in the `_nino` command
```
Comment (WalshyDev) [inline changelog.md]: You can put these two into one bullet point

File: changelog.md [hunk 2]
```diff
@@ -6,12 +6,24 @@
...
+ - Jumbo
+	 - Ever wanted bigger emojies? Now you can with the jumbo command.
```
Comment (WalshyDev) [inline changelog.md]: *emotes

File: changelog.md [hunk 3]
```diff
@@ -6,12 +6,24 @@
...
+	 - Supports up the 5 emojie at once.
+	 - Gives info on emojies
```
Comment (WalshyDev) [inline changelog.md]: Combine these two and emotes.

File: changelog.md [hunk 4]
```diff
@@ -6,12 +6,24 @@
...
+	 - Note: Only works on custom emojies
```
Comment (WalshyDev) [inline changelog.md]: It only works for emotes currently, emotes are ones uploaded by guilds not the default emojis.

File: permissions.md [hunk 5]
```diff
@@ -1,11 +1,12 @@
-# Permissions
+﻿# Permissions
```
Comment (WalshyDev) [inline permissions.md:1]: What changed here?

File: permissions.md [hunk 6]
```diff
@@ -1,11 +1,12 @@
...
 * We have introduced negation permissions meaning by putting a `-` before a permission will **deny** users access to the permission. This works in terms of group hierarchy; groups at the top will override groups at the bottom. User permissions will also immediately override group permissions.
 * As mentioned before, groups now follow a hierarchy (Groups at the top being more important). To allow for this order to be changed we have introduced the command `_perms group <group> move <position>` to move the group up and down the chain. This order can bee seen by doing `_perms groups`!
 * We have finally added the ability to do full and accurate permission validation meaning you will never be mistake on a permission again!
+* Added group cloning and renaming. Cloning copes the group and all it's perms (Doesn't copy users) to make making similar groups easier. To clone a group just do `_permissions group <group> clone <new group>`And to rename just do `_permissions group <group> rename <new name>`
```
Comment (WalshyDev) [inline permissions.md]: *copies
*to make similar groups easier
*<new_group> (also add spacing after the code end, plus lowercase the a)
*<new_name>

**Review by WalshyDev — CHANGES_REQUESTED** [review]

File: permissions.md (hunk 5 above)
Comment (weeryan17) [reply permissions.md:1]: idk

**Pull request merged**